src/loadgenerator/locustfile.py

```diff
@@ -64,6 +69,14 @@
 URLLib3Instrumentor().instrument()
 logging.info("Instrumentation complete")
 
+# Initialize Flagd provider
+api.set_provider(FlagdProvider(host=os.environ.get('FLAGD_HOST', 'flagd'), port=os.environ.get('FLAGD_PORT', 8013)))
```

Comment:
Please use this: https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/recommendationservice/recommendation_server.py#L134

Reply:
Hey, not sure what you mean by this, is it not the same?

Reply:
lol, totally missed that